### DISCUSSION TITLE
How do I set up a user manager automatically.

### DISCUSSION THREAD

**Discussion by pedrodgoes** [ory/.github#78] [Q&A] [answered]
How do I automatically define an administrator user? Is it possible to do it in the console? I'm using Custom UI with Ory Elements.

Note: I believe that where I say user, I mean identities.

**Comment by vinckr** [comment, marked as answer]
Hello @pedrodgoes 

How is an admin user defined in your system?

generally we think this is an authorization problem (so should be handled by Ory Permissions). But you can also use metadata in identities to implement roles (not best practice, but it works)
